zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -9299,6 +9301,16 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/ProcessInstanceCreationStartInstruction"
+        runtimeInstructions:
+          description: |
+            Runtime instructions (alpha). List of instructions that affect the runtime behavior of
+            the process instance. Refer to specific instruction types for more details.
+
+            This parameter is an alpha feature and may be subject to change
+            in future releases.
+          type: array
+          items:
+            $ref: "#/components/schemas/ProcessInstanceCreationRuntimeInstruction"
```

Comment:
ℹ️ @megglos, @MaxTru, and @felix-mueller for awareness. We might even need to flag alpha states for public API endpoints' parameters. It wouldn't make the whole endpoint an alpha feature, just specific parts of it 🙈

Reply:
We need to make this alpha because of the limitations of the current implementation. As part of this feature, it is impossible to resume a suspended instance, and it may not even be fully recoverable in the future when such functionality is implemented. The behavior is likely to change with https://github.com/camunda/product-hub/issues/1713.

Reply:
I don't understand this. 
My understanding:
- For 8.8 we aim to have only Suspension. 
- Later we want to add Resume, but probably we won't get to it in 8.8. 
- Suspension on its own is useful mainly for testing / development, so it's fine that we dont support resume yet.

Why can we not design the API in a way that we can keep it the same when we introduce "resume"?

Reply:
The API will not change when we introduce "resume", but process instances suspended under the current implementation will be in a potentially non-recoverable state due to how suspension is implemented now, even when "resume" is supported. Here's some more context about this: [Slack](https://camunda.slack.com/archives/C08GZ9P3XT4/p1749815731815699)